2024-04-25 16:00:01,474 INFO     asc2bin        /home/data/SP2401/raw/bx992/sp2024_115_55038.gph
2024-04-25 16:00:01,475 INFO     asc2bin        Ngood = 0   Nbad = 0
2024-04-25 16:00:02,459 INFO     asc2bin        /home/data/SP2401/raw/gpshead/sp2024_115_55038.hdg
2024-04-25 16:00:02,459 INFO     asc2bin        Ngood = 0   Nbad = 0
2024-04-25 16:00:02,465 INFO     asc2bin        /home/data/SP2401/raw/gp170b/sp2024_115_55038.gps
2024-04-25 16:00:02,466 INFO     asc2bin        Ngood = 0   Nbad = 0
2024-04-25 16:00:02,470 INFO     asc2bin        /home/data/SP2401/raw/bx992/sp2024_115_55038.gph
2024-04-25 16:00:02,470 INFO     asc2bin        Ngood = 0   Nbad = 0
2024-04-25 16:00:02,472 INFO     asc2bin        /home/data/SP2401/raw/gp170a/sp2024_115_55038.gps
2024-04-25 16:00:02,472 INFO     asc2bin        Ngood = 0   Nbad = 0
2024-04-25 18:11:24,426 INFO     asc2bin        /home/data/SP2401/raw/bx992/sp2024_115_55038.gph
2024-04-25 18:11:24,426 INFO     asc2bin        /home/data/SP2401/raw/bx992/sp2024_115_55038.gph
2024-04-25 18:11:24,427 INFO     asc2bin        Ngood = 0   Nbad = 0
2024-04-25 18:11:24,428 INFO     asc2bin        Ngood = 0   Nbad = 0
2024-04-25 18:11:24,429 INFO     asc2bin        /home/data/SP2401/raw/gp170a/sp2024_115_55038.gps
2024-04-25 18:11:24,430 INFO     asc2bin        /home/data/SP2401/raw/bx992/sp2024_115_57600.gph
2024-04-25 18:11:24,431 INFO     asc2bin        /home/data/SP2401/raw/gpshead/sp2024_115_55038.hdg
2024-04-25 18:11:24,431 INFO     asc2bin        /home/data/SP2401/raw/bx992/sp2024_115_57600.gph
2024-04-25 18:11:24,433 INFO     asc2bin        Ngood = 0   Nbad = 0
2024-04-25 18:11:24,434 INFO     asc2bin        Ngood = 0   Nbad = 0
2024-04-25 18:11:24,434 INFO     asc2bin        /home/data/SP2401/raw/gp170b/sp2024_115_55038.gps
2024-04-25 18:11:24,434 INFO     asc2bin        Ngood = 0   Nbad = 0
2024-04-25 18:11:24,434 INFO     asc2bin        Ngood = 0   Nbad = 0
2024-04-25 18:11:24,436 INFO     asc2bin        /home/data/SP2401/raw/gp170a/sp2024_115_57600.gps
2024-04-25 18:11:24,438 INFO     asc2bin        /home/data/SP2401/raw/gpshead/sp2024_115_57600.hdg
2024-04-25 18:11:24,439 INFO     asc2bin        Ngood = 0   Nbad = 0
2024-04-25 18:11:24,439 INFO     asc2bin        Ngood = 0   Nbad = 0
2024-04-25 18:11:24,440 INFO     asc2bin        Ngood = 0   Nbad = 0
2024-04-25 18:11:24,460 INFO     asc2bin        /home/data/SP2401/raw/gp170b/sp2024_115_57600.gps
2024-04-25 18:11:24,460 INFO     asc2bin        Ngood = 0   Nbad = 0
2024-04-25 19:14:35,516 ERROR    asc2bin        in make_record
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/asc2bin.py", line 474, in make_record
    rec2 = self.get_field(datastring)
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/msg.py", line 273, in get_degrees
    return [float(str_.split(b',')[1]) ]
ValueError: could not convert string to float: ''
2024-04-25 19:14:35,608 WARNING  asc2bin        Rejected /home/data/SP2401/raw/bx992/sp2024_115_65482.gph gph
   b'$PYRTM,2024,115.801799762558,6.911447172697' b'$GNHDT,,\x0051'
 None

2024-04-25 19:14:36,011 ERROR    asc2bin        in make_record
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/asc2bin.py", line 474, in make_record
    rec2 = self.get_field(datastring)
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/msg.py", line 273, in get_degrees
    return [float(str_.split(b',')[1]) ]
ValueError: could not convert string to float: ''
2024-04-25 19:14:36,012 WARNING  asc2bin        Rejected /home/data/SP2401/raw/bx992/sp2024_115_65482.gph gph
   b'$PYRTM,2024,115.801805597060,6.911453007199' b'$GNHDT,,\x0051'
 None

2024-04-25 19:14:36,514 ERROR    asc2bin        in make_record
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/asc2bin.py", line 474, in make_record
    rec2 = self.get_field(datastring)
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/msg.py", line 273, in get_degrees
    return [float(str_.split(b',')[1]) ]
ValueError: could not convert string to float: ''
2024-04-25 19:14:36,514 WARNING  asc2bin        Rejected /home/data/SP2401/raw/bx992/sp2024_115_65482.gph gph
   b'$PYRTM,2024,115.801811337141,6.911458747280' b'$GNHDT,,\x0051'
 None

2024-04-25 19:14:37,016 ERROR    asc2bin        in make_record
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/asc2bin.py", line 474, in make_record
    rec2 = self.get_field(datastring)
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/msg.py", line 273, in get_degrees
    return [float(str_.split(b',')[1]) ]
ValueError: could not convert string to float: ''
2024-04-25 19:14:37,016 WARNING  asc2bin        Rejected /home/data/SP2401/raw/bx992/sp2024_115_65482.gph gph
   b'$PYRTM,2024,115.801817172512,6.911464582650' b'$GNHDT,,\x0051'
 None

2024-04-25 19:14:37,518 ERROR    asc2bin        in make_record
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/asc2bin.py", line 474, in make_record
    rec2 = self.get_field(datastring)
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/msg.py", line 273, in get_degrees
    return [float(str_.split(b',')[1]) ]
ValueError: could not convert string to float: ''
2024-04-25 19:14:37,518 WARNING  asc2bin        Rejected /home/data/SP2401/raw/bx992/sp2024_115_65482.gph gph
   b'$PYRTM,2024,115.801822911493,6.911470321632' b'$GNHDT,,\x0051'
 None

2024-04-25 19:14:38,021 ERROR    asc2bin        in make_record
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/asc2bin.py", line 474, in make_record
    rec2 = self.get_field(datastring)
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/msg.py", line 273, in get_degrees
    return [float(str_.split(b',')[1]) ]
ValueError: could not convert string to float: ''
2024-04-25 19:14:38,022 WARNING  asc2bin        Rejected /home/data/SP2401/raw/bx992/sp2024_115_65482.gph gph
   b'$PYRTM,2024,115.801828747778,6.911476157917' b'$GNHDT,,\x0051'
 None

2024-04-25 19:14:38,523 ERROR    asc2bin        in make_record
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/asc2bin.py", line 474, in make_record
    rec2 = self.get_field(datastring)
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/msg.py", line 273, in get_degrees
    return [float(str_.split(b',')[1]) ]
ValueError: could not convert string to float: ''
2024-04-25 19:14:38,524 WARNING  asc2bin        Rejected /home/data/SP2401/raw/bx992/sp2024_115_65482.gph gph
   b'$PYRTM,2024,115.801834490486,6.911481900625' b'$GNHDT,,\x0051'
 None

2024-04-25 19:14:39,026 ERROR    asc2bin        in make_record
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/asc2bin.py", line 474, in make_record
    rec2 = self.get_field(datastring)
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/msg.py", line 273, in get_degrees
    return [float(str_.split(b',')[1]) ]
ValueError: could not convert string to float: ''
2024-04-25 19:14:39,027 WARNING  asc2bin        Rejected /home/data/SP2401/raw/bx992/sp2024_115_65482.gph gph
   b'$PYRTM,2024,115.801840319757,6.911487729896' b'$GNHDT,,\x0051'
 None

2024-04-25 19:14:39,530 ERROR    asc2bin        in make_record
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/asc2bin.py", line 474, in make_record
    rec2 = self.get_field(datastring)
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/msg.py", line 273, in get_degrees
    return [float(str_.split(b',')[1]) ]
ValueError: could not convert string to float: ''
2024-04-25 19:14:39,530 WARNING  asc2bin        Rejected /home/data/SP2401/raw/bx992/sp2024_115_65482.gph gph
   b'$PYRTM,2024,115.801846062627,6.911493472766' b'$GNHDT,,\x0051'
 None

2024-04-25 19:59:26,728 ERROR    asc2bin        in make_record
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/asc2bin.py", line 474, in make_record
    rec2 = self.get_field(datastring)
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/msg.py", line 273, in get_degrees
    return [float(str_.split(b',')[1]) ]
ValueError: could not convert string to float: ''
2024-04-25 19:59:26,734 WARNING  asc2bin        Rejected /home/data/SP2401/raw/bx992/sp2024_115_65482.gph gph
   b'$PYRTM,2024,115.801799762558,6.911447172697' b'$GNHDT,,\x0051'
 None

2024-04-25 19:59:26,739 ERROR    asc2bin        in make_record
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/asc2bin.py", line 474, in make_record
    rec2 = self.get_field(datastring)
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/msg.py", line 273, in get_degrees
    return [float(str_.split(b',')[1]) ]
ValueError: could not convert string to float: ''
2024-04-25 19:59:26,755 WARNING  asc2bin        Rejected /home/data/SP2401/raw/bx992/sp2024_115_65482.gph gph
   b'$PYRTM,2024,115.801805597060,6.911453007199' b'$GNHDT,,\x0051'
 None

2024-04-25 19:59:26,761 ERROR    asc2bin        in make_record
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/asc2bin.py", line 474, in make_record
    rec2 = self.get_field(datastring)
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/msg.py", line 273, in get_degrees
    return [float(str_.split(b',')[1]) ]
ValueError: could not convert string to float: ''
2024-04-25 19:59:26,787 WARNING  asc2bin        Rejected /home/data/SP2401/raw/bx992/sp2024_115_65482.gph gph
   b'$PYRTM,2024,115.801811337141,6.911458747280' b'$GNHDT,,\x0051'
 None

2024-04-25 19:59:26,798 ERROR    asc2bin        in make_record
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/asc2bin.py", line 474, in make_record
    rec2 = self.get_field(datastring)
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/msg.py", line 273, in get_degrees
    return [float(str_.split(b',')[1]) ]
ValueError: could not convert string to float: ''
2024-04-25 19:59:26,803 WARNING  asc2bin        Rejected /home/data/SP2401/raw/bx992/sp2024_115_65482.gph gph
   b'$PYRTM,2024,115.801817172512,6.911464582650' b'$GNHDT,,\x0051'
 None

2024-04-25 19:59:26,809 ERROR    asc2bin        in make_record
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/asc2bin.py", line 474, in make_record
    rec2 = self.get_field(datastring)
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/msg.py", line 273, in get_degrees
    return [float(str_.split(b',')[1]) ]
ValueError: could not convert string to float: ''
2024-04-25 19:59:26,825 WARNING  asc2bin        Rejected /home/data/SP2401/raw/bx992/sp2024_115_65482.gph gph
   b'$PYRTM,2024,115.801822911493,6.911470321632' b'$GNHDT,,\x0051'
 None

2024-04-25 19:59:26,831 ERROR    asc2bin        in make_record
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/asc2bin.py", line 474, in make_record
    rec2 = self.get_field(datastring)
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/msg.py", line 273, in get_degrees
    return [float(str_.split(b',')[1]) ]
ValueError: could not convert string to float: ''
2024-04-25 19:59:26,831 WARNING  asc2bin        Rejected /home/data/SP2401/raw/bx992/sp2024_115_65482.gph gph
   b'$PYRTM,2024,115.801828747778,6.911476157917' b'$GNHDT,,\x0051'
 None

2024-04-25 19:59:26,831 ERROR    asc2bin        in make_record
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/asc2bin.py", line 474, in make_record
    rec2 = self.get_field(datastring)
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/msg.py", line 273, in get_degrees
    return [float(str_.split(b',')[1]) ]
ValueError: could not convert string to float: ''
2024-04-25 19:59:26,831 WARNING  asc2bin        Rejected /home/data/SP2401/raw/bx992/sp2024_115_65482.gph gph
   b'$PYRTM,2024,115.801834490486,6.911481900625' b'$GNHDT,,\x0051'
 None

2024-04-25 19:59:26,831 ERROR    asc2bin        in make_record
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/asc2bin.py", line 474, in make_record
    rec2 = self.get_field(datastring)
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/msg.py", line 273, in get_degrees
    return [float(str_.split(b',')[1]) ]
ValueError: could not convert string to float: ''
2024-04-25 19:59:26,831 WARNING  asc2bin        Rejected /home/data/SP2401/raw/bx992/sp2024_115_65482.gph gph
   b'$PYRTM,2024,115.801840319757,6.911487729896' b'$GNHDT,,\x0051'
 None

2024-04-25 19:59:26,831 ERROR    asc2bin        in make_record
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/asc2bin.py", line 474, in make_record
    rec2 = self.get_field(datastring)
  File "/usr/local/lib/python3.8/dist-packages/pycurrents-0.0.0-py3.8-linux-x86_64.egg/pycurrents/data/nmea/msg.py", line 273, in get_degrees
    return [float(str_.split(b',')[1]) ]
ValueError: could not convert string to float: ''
2024-04-25 19:59:26,831 WARNING  asc2bin        Rejected /home/data/SP2401/raw/bx992/sp2024_115_65482.gph gph
   b'$PYRTM,2024,115.801846062627,6.911493472766' b'$GNHDT,,\x0051'
 None

